piccolo_admin/endpoints.py

```diff
@@ -486,6 +505,11 @@ def __init__(
             app=StaticFiles(directory=os.path.join(ASSET_PATH, "js")),
         )
 
+        self.mount(
+            path="/static",
```

Comment:
I would mount this at `/media/` instead - usually static refers to an application's own files, and media refers to uploaded files.

Reply:
You're right, I'll change that.